modules/imgproc/test/test_drawing.cpp

```diff
@@ -913,4 +913,11 @@ INSTANTIATE_TEST_CASE_P(
     )
 );
 
+TEST(Drawing, DISABLED_circle_overflow)
```

Comment:
Why `DISABLED_`?

Reply:
It takes significant time. I decided to disable it by default. Should I return int back?

Reply:
`applyTestTag(CV_TEST_TAG_VERYLONG);` or at least add comment.

Reply:
Enabled test with `CV_TEST_TAG_VERYLONG` option.